drivers/pure/flasharray/networkInterfaces.go

```diff
@@ -0,0 +1,37 @@
+package flasharray
+
+type Eth struct {
+	Subtype       string `json:"subtype"`
+	Subinterfaces []int  `json:"subinterfaces"`
+	Address       string `json:"address"`
+	MacAddress    string `json:"mac_address"`
+	Mtu           int    `json:"mtu"`
+	Gateway       string `json:"gateway"`
+	Vlan          *int   `json:"vlan"`
+	Netmask       string `json:"netmask"`
+	Subnet        Subnet `json:"subnet"`
+}
+type Subnet struct {
+	IP   string `json:"ip"`
+	Mask string `json:"mask"`
+}
+type Fc struct {
```

Comment:
[nit] We also have to worry about NVMe interfaces. I doubt we'll be testing those *yet* but in the future we might. I'm fine leaving it out for now because I know it's hard to get NVMe setups.

Reply:
Currently leaving it as is, as per discussion